### DISCUSSION TITLE
how can I Force user to turn on the Internet on the phone

### DISCUSSION THREAD

**Discussion by jeuxdaily** [distriqt/ANE-IronSource#56] [General]
I publish my app in adobe animate cc with ads

normally the ads are displayed with internet connection.

in this case how can I force the user to connect internet to play my app so that the ads are displayed.

if the user does not connect to the internet the application stops.

thanks.

**Comment by marchbold** [comment]
Hi, There is no functionality like that in this extension. You need to use a network connectivity test and then handle it in your application appropriately. 

You could use the NetworkInfo ANE 

https://docs.airnativeextensions.com/docs/networkinfo/network-change-monitoring

or just standard AIR functionality.

**Reply by jeuxdaily** [reply]
thanks 
can You give an examlpe

**Reply by marchbold** [reply]
```actionscript
NetworkInfo.networkInfo.addEventListener( 
        NetworkInfoEvent.CHANGE,
        networkChangeHandler 
    );

function networkChangeHandler( event:NetworkInfoEvent ):void
{
    if (!NetworkInfo.networkInfo.isReachable())
    {
        trace( "network not available" );
    }
}

```

**Comment by jeuxdaily** [comment]
hi
I have this problem

1046: Type was not found or was not a compile-time constant: NetworkInfoEvent.

**Reply by marchbold** [reply]
Did you add the extension and package it with your application?

**Comment by jeuxdaily** [comment]
I could not find the download file on the site
Can You put NetworkInfo ANE link
thanks

**Reply by marchbold** [reply]
https://airnativeextensions.com/extension/com.distriqt.NetworkInfo
